MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,961 @@
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [DataType(DataType.Time)]
+    [Display(Name = "Birth Time")]
+    public TimeSpan? BirthTime { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+`images`
+
+**Customizing format in time editor**
+
+In the [TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows), short time will be shown by default. You can change the applied format by setting the [Format]`Link`property in [DataFormTimeItem]`Link`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "BirthTime" && e.DataFormItem is DataFormTimeItem timeItem)
+        {
+            timeItem.Format = "HH:mm";
+        }
+    } 
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## CheckBox editor
+
+In [CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0) editor, the [CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0) control is loaded. By default, for `bool` data type property, the [CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0) editor will be loaded in data form.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [Display(Name = "Is Billable")]
+    public bool IsBillable { get; set; } = true;
+
+    [Display(Name = "Registered Member")]
+    public bool RegisteredMember { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+**Visual states of CheckBox**
+
+CheckBox support two visual states.
+  - Checked
+  - UnChecked
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [Display(Name = "Is Billable")]
+    public bool IsBillable { get; set; } = true;
+
+    [Display(Name = "Registered Member")]
+    public bool RegisteredMember { get; set; } = false;
+
+{% endhighlight %}
+{% endtabs %}
+		
+## Switch Editor
+
+In switch editor, [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) is loaded, and DataForm [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) editor supports bool data type property.
+
+To add [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) editor in DataForm, register the editor as [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) for the required property using the [RegisterEditor]`Link` method.
```

Comment:
register the editor as [Switch] - register the editor as DataFormEditorType.Switch

Here we mention about registering so remove link of Switch

Reply:
DataFormEditorType.Switch added